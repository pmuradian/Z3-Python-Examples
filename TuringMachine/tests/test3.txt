3 3 3
a b c
i j f
i
f
a i b j 1
a j b j -1
b j a f 1
4
a a a a
