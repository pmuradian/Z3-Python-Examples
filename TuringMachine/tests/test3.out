YES

a|i a|  a|  a|
b|  a|j a|  a|
b|j b|  a|  a|
a|  b|f a|  a|?â‰¥YESa|i a| a| a| b| a|j a| a| b|j b| a| a| a| b|f a| a| YESa|i a| a| a| b| a|j a| a| b|j b| a| a| a| b|f a| a| YESa|i a| a| a| b| a|j a| a| b|j b| a| a| a| b|f a| a| YESa|i a| a| a| b| a|j a| a| b|j b| a| a| a| b|f a| a| 